Python/suggestions.c

```diff
@@ -43,7 +43,7 @@ levenshtein_distance(const char *a, size_t a_size,
                      const char *b, size_t b_size,
                      size_t max_cost)
 {
-    static size_t buffer[MAX_STRING_SIZE];
+    size_t buffer[MAX_STRING_SIZE] = {0};
```

Comment:
Can't we leave off the `= {0}`? That way, this is just bumping the stack pointer, no initialization necessary.